src/notification/model.rs

```diff
@@ -76,10 +76,17 @@ impl Notifications {
                     JOIN event_log on event_log.id = e.event_log_id
                     JOIN uuid uuid1 on uuid1.id = event_log.uuid_id
                     LEFT JOIN entity entity1 on entity1.id = event_log.uuid_id
+                    LEFT JOIN event_parameter ON event_parameter.log_id = event_log.id
+                    LEFT JOIN event_parameter_uuid ON
+                      event_parameter_uuid.event_parameter_id = event_parameter.id
+                    LEFT JOIN uuid uuid2 on uuid2.id = event_parameter_uuid.uuid_id
+                    LEFT JOIN entity entity2 on entity2.id = event_parameter_uuid.uuid_id
                     WHERE n.user_id = ?
-                      AND uuid1.discriminator != "attachment"
-                      AND uuid1.discriminator != "blogPost"
+                      AND uuid1.discriminator NOT IN ("attachment", "blogPost")
+                      AND (uuid2.discriminator IS NULL OR
+                        uuid2.discriminator NOT IN ("attachment", "blogPost"))
                       AND (entity1.type_id IS NULL OR entity1.type_id IN (1,2,3,4,5,6,7,8,49,50))
+                      AND (entity2.type_id IS NULL OR entity2.type_id IN (1,2,3,4,5,6,7,8,49,50))
```

Comment:
A bit hard to read & understand (SQL ftw I guess...). Does this check whether UUIDs that occur in the event params also have the right type? I'm not entirely sure whether this works correctly for events with two UUID parameters since I think that you only check the first UUID parameter (because of `LEFT JOIN`).

Reply:
> Does this check whether UUIDs that occur in the event params also have the right type?

Yes

>  I'm not entirely sure whether this works correctly for events with two UUID parameters since I think that you only check the first UUID parameter (because of `LEFT JOIN`).

Are there any? I thought there would never be such an event...

However: It is possible that event_parameter_uuid is an unsupported uuid type when event_log.uuid is supported?! I check this...

Reply:
Okay, I have found an event type where it is necessary to check the uuid_paramater: create_thread  https://github.com/serlo/serlo.org-database-layer/blob/main/src/event/model/create_thread.rs#L19-L31 We might have a thread on an unsupported entity / uuid.

Reply:
Okay, I have added code to deduplicate notifications when they have multiple event_parameter_uuid -> however I haven't seen an event with multiple event_paramater_uuid yet...